enterprise/app/tap/tap.tsx

```diff
@@ -104,6 +112,13 @@ export default class TapComponent extends React.Component<Props, State> {
     router.replaceParams({ repo });
   }
 
+  branchInputRef = React.createRef<HTMLInputElement>();
```

Comment:
i feel like this style (field near relevant function decl) is more of a go thing? i dunno, i'd put it up with the other class fields.

Reply:
Done - was just being lazy at the time and forgot to move this up